internal/controller/postgrescluster/instance_test.go

```diff
@@ -261,6 +262,176 @@ func TestNewObservedInstances(t *testing.T) {
 	})
 }
 
+func TestWritablePod(t *testing.T) {
+	env, cc, config := setupTestEnv(t, ControllerName)
```

Comment:
🔧 This test doesn't use the API, reconciler, nor manager. Setting up envtest just slows things down.